server/dataone_register.py

```diff
@@ -126,22 +149,28 @@ def find_resource_pid(pid):
         "Multiple resource maps were for the data package, which isn't supported.")
 
 
-def find_nonobsolete_resmaps(pids):
+def find_nonobsolete_resmaps(pids, base_url):
     """
     Given one or more resource map pids, returns the ones that are not obsoleted
     by any other Object.
     This is done by querying the Solr index with the -obsoletedBy:* query param
+
+    :param pids: The pids that are checked
+    :param base_url: A coordinating node that will be used to check
+    :return:
     """
 
+    logger.debug('Entered find_nonobsolete_resmaps')
     result = query(
         "identifier:(\"{}\")+AND+-obsoletedBy:*".format("\" OR \"".join(pids),
-                                                        fields="identifier")
-    )
+                                                        url=base_url,
+                                                        fields="identifier"))
     result_len = int(result['response']['numFound'])
 
     if result_len == 0:
         raise RestException('No results were found for identifier(s): {}.'.format(", ".join(pids)))
 
+    logger.debug('Leaving find_nonobsolete_resmaps')
```

Comment:
Does the rest of the wholetale plugin use logging statements like these (entering, exiting)? It's super handy when the logfile is somewhat consistent in terms of verbosity message structure. These types of logging statements might be too noisy since they're as useful as something like "Found nonobsoleted resource map from {pids} of {found_pid}"

Reply:
It's not done like that elsewhere, but I've found it to be really helpful as a trace. I wish Girder had a `trace` level logger because these statements would be more appropriate there. 

Do you think if it would be too messy if I comment them out? I'd still like to get a trace when debugging.